pkg/api/frontendsettings.go

```diff
@@ -551,7 +551,7 @@ func (hs *HTTPServer) getFSDataSources(c *contextmodel.ReqContext, availablePlug
 			dsDTO.Database = ds.Database
 		}
 
-		if ds.Type == datasources.DS_PROMETHEUS {
+		if ds.Type == datasources.DS_PROMETHEUS || ds.Type == datasources.DS_AMAZON_PROMETHEUS || ds.Type == datasources.DS_AZURE_PROMETHEUS {
```

Comment:
Can we move the new `isPrometheusCompatible()` function introduced in https://github.com/grafana/grafana/pull/103716/files#diff-822280e15014acfd5ca557137b0a0c5d8ecec0ddac2c65a646e8729aafa92ac5R44 and use it here also?

Reply:
I'll follow up with this in a separate PR